src/Entity/EntityId.php

```diff
@@ -17,6 +18,30 @@
 
 	protected $serialization;
 
+	const PATTERN = '/^:?(\w+:)*[^:]+\z/';
```

Comment:
The regex in ItemId and PropertyId do not allow `:a:Q1`, but this here does. One must be wrong.